cognite_toolkit/cdf_tk/load.py

```diff
@@ -226,6 +226,15 @@ def get_id(self, item: DataSet) -> str:
     def delete(self, ids: Sequence[str]) -> None:
         raise NotImplementedError("CDF does not support deleting data sets.")
 
+    def create(
+        self, items: Sequence[T_Resource], ToolGlobals: CDFToolConfig, drop: bool, filepath: Path
+    ) -> T_ResourceList | None:
```

Comment:
```suggestion
    ) -> T_Resource | T_ResourceList | None:
```
You only return a single in file, the `drop_load_resources` function must be updated to ensure it supports it.

Reply:
Right.

Reply:
Hm, this is datasets loader?